x-pack/plugins/apm/public/components/shared/PropertiesTable/NestedKeyValueTable.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import {
+  colors,
+  fontFamilyCode,
+  fontSizes,
+  units
+} from '../../../style/variables';
+
+interface StringMap<T> {
+  [key: string]: T;
+}
+
+type KeySorter = (data: StringMap<any>, parentKey?: string) => string[];
+
+const Table = styled.table`
+  font-family: ${fontFamilyCode};
+  font-size: ${fontSizes.small};
+  width: 100%;
+`;
+
+const Row = styled.tr`
+  border-bottom: 1px solid ${colors.gray4};
+  &:last-child {
+    border: 0;
+  }
+`;
+
+const Cell = styled.td`
+  vertical-align: top;
+  padding: ${units.half}px 0;
+
+  ${Row}:first-child> & {
+    padding-top: 0;
+  }
+
+  ${Row}:last-child> & {
+    padding-bottom: 0;
+  }
+
+  &:first-child {
+    width: 300px;
```

Comment:
I try to use [units](https://github.com/jasonrhodes/kibana/blob/ba351d805f58f721f25c96f966d26b55b162d5d6/x-pack/plugins/apm/public/style/variables.ts#L10-L20) everywhere, to ensure a grid-like consistency for width, height, margin and padding.

In this case I'd make it 20 units (if I recall correctly, 300px is just an arbitrary number)

Reply:
Can you explain how this works for something like 20?

Reply:
Something like this
```
width: ${unit * 20};
```